src/shared/components/top-header/top-header.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="flex flex-row px-40 py-2 bg-grey-100 text-grey-600">
+    <svg-icon [src]="'localization-icon'"></svg-icon>
+  <select (change)="onChange($event)">
+    <option disabled selected>Language</option>
```

Comment:
just for styling purposes, could you try adding a bit different, for example more pale color to the selected disabled option, for better UI